packages/p2p/src/utils/websocket.js

```diff
@@ -102,6 +102,7 @@ const getModifiedP2POrder = response => {
     const price_rate = +response.rate;
     const transaction_amount = +response.price;
     const payment_method = map_payment_method.bank_transfer; // TODO: [p2p-replace-with-api] add payment method to order details once API has it
+    const chat_channel_url = response.chat_channel_url;
```

Comment:
Can be destructured:

```suggestion
    const { chat_channel_url } = response;
```